frontend/public/style/_common.scss

```diff
@@ -25,8 +25,8 @@ $co-external-link-padding-right: 15px;
   flex-wrap: wrap;
   justify-content: space-between;
   &--alt {
-    margin-left: 0;
-    margin-right: 0;
+    margin-left: 0 !important;
```

Comment:
Reluctant to add another `!important`, especially since this class is used on a number of pages.

Reply:
Thanks for the explanation. Could you extract that change into its own PR? I think it warrants separate consideration.

Reply:
I only see Rules using that css selector.   Here we see it doesn't actually do the 0 margins:
![margins_not_zero](https://user-images.githubusercontent.com/12733153/67214161-3eec8f00-f3ed-11e9-9e14-3ff294341258.png)

With the `!important` we do see 0 margins.  This is important as with 0 margins the [Create Receiver] button does left-align correctly with section title and table.  It wasn't as apparent in the Rules usage as there was no left button:
![margins_are_zero](https://user-images.githubusercontent.com/12733153/67214268-71968780-f3ed-11e9-8e54-a835d6ce7950.png)

Do you really want a separate PR?  I don't see it as a global change  -thanks

Reply:
👍 Thanks

I was wrong about `co-m-pane__filter-bar--alt` being used on a number of pages. It's actually only used on one other page (the RBAC role details page).